paictl.py

```diff
@@ -502,6 +505,51 @@ def update_external_config(self, args):
         external_conf_update.update_latest_external_configuration()
 
 
+class Utility(SubCmd):
```

Comment:
Should we put a new command to paictl?
It looks like won't work for the platform we support. What about leave it as a separately script?

Reply:
I prefer to manage them in one tool like paictl. And why it won't work for the platform we support?

Reply:
My concern is that: 
1. paictl should focus on the major features of PAI.
2. We could provide associated tools for convenient, and release it as a separately script. In case there are qualified thirdparty tools, we can replace it.

There are plenty of tool for copying or uploading files, and we are not completing with them.

Reply:
1. And you also should find a tool which could leverage our machine-list. So that, user could avoid to write another machine-list to do it.  
2. Our solution is based on paramiko.

Reply:
Maybe we could rise an issue to track this idea.

Reply:
Hi @ydye,

It's about we should be carefully when we are adding a new command.
We frequently update commands, user will be annoyed .
I'm against adding a new 'utils' sub command, because we are not sure it's really needed.

In the same time, I recommend to provide it in a more flex format, for example, a standalone script.